### PR TITLE
@cavvia => Resolve AB test for market insights and roll it out to everbody

### PR DESCRIPTION
**Pull request by mzikherman**
Apparently the test finished relatively flat (50/50) in its results...which is a win for us! Score! We'll take it.

### CODE REVIEW

File: desktop/apps/artwork/components/artists/helpers.coffee [hunk 1]
```diff
@@ -15,24 +15,21 @@ module.exports =
...
     has: (section) ->
       switch section
         when 'biography'
           artist.blurb || artist.bio || artist.biography_blurb?.text != ""
         when 'exhibition_highlights'
-          if hasMarketDataEnabled
-            artist.exhibition_highlights? && artist.exhibition_highlights.length > 0
-          else
-            artist.exhibition_highlights? && artist.exhibition_highlights.length > 15
+          artist.exhibition_highlights?.length > 0
```
Comment (mzikherman) [inline desktop/apps/artwork/components/artists/helpers.coffee]: b/c we're using `?` existential operator we don't need two separate conditionals

File: desktop/apps/artwork/components/artists/templates/biography.jade [hunk 2]
```diff
@@ -15,9 +15,3 @@
     p= artist.bio.replace('born', 'b.')
 
 div(id="market-insights-container-" + artist._id)
-
-unless sd.ARTIST_MARKET_DATA === 'experiment'
```
Comment (mzikherman) [inline desktop/apps/artwork/components/artists/templates/biography.jade]: This removes the highlights/exhibition information that we used to display (now replaced by market insights).

It's now located in its own tab.

**Comment by mzikherman** [comment]
Bump! I think we're cool to roll out market insights to everybody? cc @madeleineb

**Comment by madeleineb** [comment]
We absolutely are! let's do it @mzikherman 😸

File: desktop/apps/artwork/components/artists/helpers.coffee (hunk 1 above)
Comment (cavvia) [reply desktop/apps/artwork/components/artists/helpers.coffee]: :+1:

**Comment by cavvia** [comment]
Apologies, this PR got lost as I was OOO last week. This looks good to me. Merging in so we can test on Staging!

**Pull request merged**